testem.js

```diff
@@ -5,8 +5,7 @@ module.exports = {
   disable_watching: true,
   framework: 'mocha',
   launch_in_ci: [
-    'Chrome',
-    'Firefox'
```

Comment:
What is the reason for this?  This is not the pattern across the rest of the `ciena-frost` organization - https://github.com/search?q=org%3Aciena-frost+Firefox&type=Code